composer.json

```diff
@@ -330,5 +328,8 @@
             "cache:clear": "symfony-cmd",
             "assets:install %PUBLIC_DIR%": "symfony-cmd"
         }
-    }
+    },
+    "repositories": [
+        { "type": "vcs", "url": "git@github.com:Zales0123/SymfonyMockerContainer.git" }
+    ]
```

Comment:
Btw, this is obviously a temporary solution before we migrate to something else or add support for Symfony 6 on the MockerContainer itself 🖖